__tests__/HomePage.test.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Homepage from "../src/pages/Homepage";
```

Comment:
Please use the module resolver for jest   [here](https://github.com/benfluleck/beth-in-froopyland/blob/develop/package.json#L46)

Reply:
Done